pkg/apis/core/types.go

```diff
@@ -2146,6 +2147,25 @@ const (
 	NodeSelectorOpLt           NodeSelectorOperator = "Lt"
 )
 
+// A topology selector term represents the result of label queries over a set of nodes.
```

Comment:
@saad-ali suggested that this could be potentially utilized for labels that are on other objects besides nodes.  So we should remove mentions of node in the comments here, but we can leave it on StorageClass.allowedTopologies.